cassandra-migration-etl/form-service-migration/migrate.js

```diff
@@ -0,0 +1,105 @@
+// Form service migration script
+// Accepted arguments
+// 1. contactPoints: IP with port of DB
+// 2. username: username for DB // optional
+// 3. password: password for DB // optional
+// example: node migration.js 11.7.1.7:9200 username password
+
+const cassandra = require('cassandra-driver');
+let cassandraClientOptions = { contactPoints: [process.argv[2]] };
+if(process.argv[3] && process.argv[4]){
+  cassandraClientOptions.authProvider = new cassandra.auth.PlainTextAuthProvider(process.argv[3], process.argv[4]);
+}
+console.log('connecting to DB with', process.argv[2], process.argv[3], process.argv[4]);
+const client = new cassandra.Client(cassandraClientOptions);
+
+let transformed_data = [];
+let dest_obj = {
+  root_org: undefined,
```

Comment:
Use camel case for column names. e.g. rootOrgId

Have following columns as well.

createdOn timestamp, 
createdBy text, 
lastUpdatedOn timestamp, 
lastUpdatedBy text

Reply:
[Cassandra will by default lowercase all identifier names unless quoted](https://docs.datastax.com/en/cql/3.1/cql/cql_reference/ucase-lcase_r.html). So for the column names, i'm keeping it in snake case.